src/components/NavigationLIstAccordion.tsx

```diff
@@ -0,0 +1,56 @@
+import { ReactNode } from 'react';
+import { useHistory } from 'react-router-dom';
+import { Accordion, AccordionDetails, AccordionProps, AccordionSummary, Box, Typography } from '@mui/material';
+import ExpandMoreIcon from '@mui/icons-material/ExpandMore';
+
+interface NavigationListAccordionProps extends AccordionProps {
+  title: string;
+  startIcon: ReactNode;
+  accordionPath: string;
+  defaultPath: string;
+  dataTestId?: string;
+}
+
+export const NavigationListAccordion = ({
+  title,
+  startIcon,
+  accordionPath,
+  defaultPath,
+  dataTestId,
+  children,
+}: NavigationListAccordionProps) => {
+  const history = useHistory();
+  const currentPath = history.location.pathname.replace(/\/$/, ''); // Remove trailing slash
+
+  return (
+    <Accordion
+      data-testid={dataTestId ?? ''}
+      disableGutters
+      expanded={currentPath.includes(accordionPath)}
+      elevation={0}
+      sx={{
+        mb: '0.5rem',
+        ':last-child': {
+          mt: '0.5rem',
+        },
+        '&.MuiAccordion-root.Mui-expanded': {
+          mt: 0,
+          mb: '0.5rem',
+        },
+        bgcolor: 'secondary.dark',
+      }}>
+      <AccordionSummary
+        sx={{ paddingX: '0.75rem' }}
+        expandIcon={<ExpandMoreIcon />}
```

Comment:
Usikker på hva jeg syns selv her, men føler det blir feil å vise en knapp som sier at man kan lukke accordion igjen når det ikke er mulig. Syns det blir mindre pent da, men jeg lener likevel mot å utelate den når den er åpen. Ser begge deler i skisser, så sikkert lurt å avklare med cecilie først 
```suggestion
        expandIcon={!isExpanded && <ExpandMoreIcon />}
```